test/setup.js

```diff
@@ -0,0 +1,21 @@
+import sinon from 'sinon';
+import chai from 'chai';
+import dirtyChai from 'dirty-chai';
+import sinonChai from 'sinon-chai';
+
+//
+// Sinon
+//
+global.sandbox = sinon.sandbox.create();
+
+afterEach(() => {
+  global.sandbox.restore();
+});
+
+//
+// Chai
+//
+global.expect = chai.expect;
+chai.should();
+chai.use(dirtyChai);
+chai.use(sinonChai);
```

Comment:
[dirty-chai](https://github.com/prodatakey/dirty-chai) allows for function form for terminating assertion properties:

```js
should.be.true();
// vs
should.be.true;
```

This also keeps the linter happy and just makes more sense.

[sinon-chai](https://github.com/domenic/sinon-chai) adds chai assertions for sinon mock/spy/stubs.  This makes the assertion errors much more helpful as well.

```js
spy.called.should.be.ok();
// vs
spy.should.have.been.called();
```
>throws something like: "expected spy to been called at least once, but was not called."